native_client/args.h

```diff
@@ -45,6 +47,7 @@ void PrintHelp(const char* bin)
     "	-t			Run in benchmark mode, output mfcc & inference time\n"
     "	--extended		Output string from extended metadata\n"
     "	--json			Extended output, shows word timings as JSON\n"
+    "	--stream size	Run in stream mode, output intermediate results\n"
```

Comment:
nit: alignment; also, what does `size` means ?

Reply:
Oh, my IDE has a different settings so it's aligned there.

What kind of tab size do we use in DS project? I set it to 2 as I see all C code here uses 2-space indent.

Reply:
BTW, size means how much data are fed to DS decoder in streaming mode.

E.g. if you are you running DS with audio source from a microphone or from remote HTTP client, it cannot provide you the whole data all at once, but frame by frame (aka streaming mode).